lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -35,9 +36,11 @@ export type AccountInfo = {
     };
     nativeAccountId?: string;
     authorityType?: string;
+    tenants?: string[];
 };
 
 export type ActiveAccountFilters = {
     homeAccountId: string;
     localAccountId: string;
+    tenantId: string;
```

Comment:
How does upgrade / rollback work for this addition?

Reply:
Good observation, will probably have to add the extra logic so this gets ignored by lower versions and it modernizes old entries.

Reply:
This is a requirement for merging this PR.  Also note that deployments mean that a single client could bounce back and forth repeatedly between old/new versions of MSAL during a rollout.

Reply:
I made it optional and added update logic for the cached account entities